### DISCUSSION TITLE
Cannot create region in already played area

### DISCUSSION THREAD

**Discussion by GabrielZirk** [katspaugh/wavesurfer.js#3394] [Q&A] [answered]
Hello,

I'm currently facing an issue with creating regions, when updating to wavesurfer v7. As mentioned in the title, I cannot create a new region in an area that was already played. [Here is a pen](https://codepen.io/GabrielAuphonic/pen/abXxvKQ?editors=1111) to show you this issue. Just click somewhere in the middle of the track to set the currentTime cursor, then try creating a region in an area that was already played.

As you can see in the JS field, I got the wavesurfer code from the npm page (https://www.npmjs.com/package/wavesurfer.js?activeTab=code). Scroll to the bottom to see the wavesurfer init.

Hope you can help me, I appreciate your help a lot! 
Best Gabriel

**Comment by katspaugh** [comment, marked as answer]
It's fixed in the latest version as you can see here: https://wavesurfer.xyz/examples/?regions.js

It looks like you've copy-pasted the entire code of an earlier version of wavesurfer. In CodePen, you can add external scripts via Settings.

**Reply by GabrielZirk** [reply]
Thanks a lot for your immediate answer! I will use the recent version.
